pulsar-client-api/src/main/java/org/apache/pulsar/client/api/Schema.java

```diff
@@ -426,6 +426,19 @@ default void configureSchemaInfo(String topic, String componentName,
         return DefaultImplementation.newAutoProduceSchema(schema);
     }
 
+    /**
+     * Create a schema instance that accepts a serialized Avro payload
+     * without validating it against the schema specified. 
+     * It can be useful when migrating data from existing event or message stores.
+     *
+     * @return the auto schema instance
+     * @since 2.9.0
+     * @see #AUTO_PRODUCE_BYTES()
+     */
+    static Schema<byte[]> AUTO_PRODUCE_VALIDATED_AVRO_BYTES(Schema<?> schema) {
+        return DefaultImplementation.newAutoProduceValidatedAvroSchema(schema);
```

Comment:
IIUC this "schema" is a "org.apache.avro.Schema"

so you have to declare your method as:
`static Schema<byte[]> AUTO_PRODUCE_VALIDATED_AVRO_BYTES(Object nativeAvroSchema)`
you can do the Cast inside AutoProduceValidatedAvroBytesSchema in which you will have the constructor like

```
AutoProduceValidatedAvroBytesSchema(Object schema) {
   if (! schema instanceof org.apache.avro.Schema) throw new IllegalArgumentException...
}
```

I suggest to rename the method to   `Schema<byte[]> NATIVE_AVRO(Object schema)`

Reply:
I see what you mean. will make the change.